xmodule/modulestore/tests/utils.py

```diff
@@ -137,7 +137,7 @@ def descend(parent, stack):
             for _ in range(branching):
                 child = ItemFactory.create(
                     category=xblock_type,
-                    parent_location=parent.location,
+                    parent=parent,
```

Comment:
Can you please explain in the commit message why this was change was necessary?

Reply:
When using `parent_location` instead of `parent` I get an error related with children/parent problem for split modulestore. Looking at other tests using split modulestore - `parent` argument is used only. I'll add it to commit message later.